app/assets/javascripts/admin/addon/components/admin-user-fields-form.gjs

```diff
@@ -0,0 +1,273 @@
+import Component from "@glimmer/component";
+import { cached, tracked } from "@glimmer/tracking";
+import { hash } from "@ember/helper";
+import { action } from "@ember/object";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import { schedule } from "@ember/runloop";
+import { service } from "@ember/service";
+import { eq, or } from "truth-helpers";
+import Form from "discourse/components/form";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { i18n } from "discourse-i18n";
+import ValueList from "admin/components/value-list";
+import UserField from "admin/models/user-field";
+
+export default class AdminUserFieldsForm extends Component {
+  @service dialog;
+  @service router;
+  @service adminUserFields;
+  @service adminCustomUserFields;
+
+  @tracked
+  editableDisabled =
+    this.args.userField?.requirement === "for_all_users";
+  userField;
+  originalRequirement;
+
+  constructor() {
+    super(...arguments);
+    this.originalRequirement = this.args.userField.requirement;
+  }
```

Comment:
you can probably just do:

```
originalRequirement = this.args.userField.requirement;
```

no need to put this in constructor

Reply:
I think you're right. This was left here after a hackier implementation. 😬